packages/lime-cc/files/lime-cc-client.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+trusted_ids=$(uci get lime-defaults.trusted_nodes.node_id 2>/dev/null)
+
+if [[ $? == 1 ]]; then
+    echo "please add section trusted_nodes in lime-defaults"
+    exit 1
+fi
+
+echo "checking"
+# get all recevied configs
+for config_path in /var/run/bmx7/sms/rcvdSms/*lime-defaults; do
+    # check if installed config is older then received config
+    if [[ /etc/config/lime-defaults -ot "$config_path" ]]; then
```

Comment:
IMO it would be better to compare MD5SUM or SHA256SUM instead. Cause if bmx7 is restarted files will be recreated and config will be applied. Also on each reboot the file would be new so executed again.

This sum might be stored into /etc/config/lime so the even after reboot it will be saved.

Reply:
Good idea, I'll look into that. Genereally I thought of installing a `opkg` package instead of just the `lime-defaults` to actually enable ssh-key adding or whatever is required. THis would also enable versioning without the need of "age" checks.